os/fs/driver/mtd/smart.c

```diff
@@ -433,7 +433,7 @@ struct smart_journal_entry_s {
 	uint8_t crc16[2];			/* CRC-16 for current journal data */
 	uint8_t psector[2];			/* Target Physical Sector, For MTD_ERASE, Target Physical Block */
 	uint8_t seq[2];				/* Sequence number of Journal for validation check */
-	uint8_t status;				/* Journal Staus low 4bits for state, high 4bits for type */
+	uint8_t status;				/* Journal Status low 4bits for state, high 4bits for type */
```

Comment:
I appreciate the spelling/grammatical corrections here but unfortunately those should be put in a separate commit please.
Especially since there are multiple of these corrections here.

Reply:
Pending